cli/src/commands/split.rs

```diff
@@ -77,12 +84,13 @@ You are splitting a commit in two: {}
 The diff initially shows the changes in the commit you're splitting.
 
 Adjust the right side until it shows the contents you want for the first
-(parent) commit. The remainder will be in the second commit. If you
+{} commit. The remainder will be in the {} commit. If you
 don't make any changes, then the operation will be aborted.
 ",
-        tx.format_commit_summary(&commit)
+        tx.format_commit_summary(&commit),
+        if args.siblings { "" } else { "(parent)" },
+        if args.siblings { "sibling" } else { "child" }
```

Comment:
I updated the language and made it static so that the same message is used in all cases.

I'm not up to speed on the discussion about whether to refer to things as revisions or commits, but it seems like there might be some chatter about it on Discord. Do you have a preference? I just left it as "commit" for now.

Reply:
Regarding "revision vs. commit", some people have expressed their mental model in [comments about the help text](https://github.com/martinvonz/jj/issues/3156#issuecomment-2019365692), showing the confusion.

Reply:
I'm going to leave it as "commit" for now since that's what it said before. If we do a larger rename later we can change it to revision then.